src/backend/access/transam/xlog.c

```diff
@@ -144,6 +144,7 @@ bool		XLOG_DEBUG = false;
 
 int			wal_segment_size = DEFAULT_XLOG_SEG_SIZE;
 
+bool		wal_need_seg_switch = false;
```

Comment:
Why do we need to swtich segments? I thought with the new solution we would not need to do that?

Reply:
It's a leftover, I'll remove it